playbooks/roles/backups/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+---
```

Comment:
There wasn't a good way to deploy this to our Docker customers, so I wrote a [one-off script](https://gist.github.com/mrgnr/f9d402077ee7bbfeb07dcea8823bf905) to configure backups there. See comment on [Trello card](https://trello.com/c/TTZFycID/232-16-automate-our-db-backups-31).